code/modules/projectiles/projectile/special/rocket.dm

```diff
@@ -83,7 +83,9 @@
 	/turf/closed,
 	/obj/vehicle/sealed/mecha,
 	/obj/machinery/door/,
-	/obj/machinery/door/poddoor/shutters
+	/obj/machinery/door/poddoor/shutters,
+	/obj/structure/window/,
```

Comment:
```suggestion
	/obj/structure/window,
```

Reply:
commit this whne bot is fixed to make bot green

Reply:
bot fixesd??

Reply:
doesnt that prevent subtypes from being selected